src/react-apps/applications/ux-editor/src/store/index.ts

```diff
@@ -18,8 +18,8 @@ function configureStore(initialState?: any): Store<IAppState> {
   let enhancer: any;
 
   if (process.env.NODE_ENV === 'development') {
-    const { logger } = require('redux-logger');
-    middlewares.push(logger);
+    // const { logger } = require('redux-logger');
+    // middlewares.push(logger);
```

Comment:
Should be removed if not needed, or commented back in

Reply:
Forgot about that, uncommented.